definitions.json

```diff
@@ -102,6 +106,283 @@
                 "type": "void"
             }
         },
+        "dexGeneral": {
+            "getPairByAssetId": {
+                "params": [
+                    {
+                        "name": "asset_0",
+                        "type": "AssetId"
+                    },
+                    {
+                        "name": "asset_1",
+                        "type": "AssetId"
+                    }
+                ],
+                "type": "Option<PairInfo<AccountId, NumberOrHex, AssetId>>"
+            },
+            "getAmountInPrice": {
+                "params": [
+                    {
+                        "name": "supply",
+                        "type": "AssetBalance"
```

Comment:
won't this crash serde?

Reply:
Just tested and I think not, although I'm not sure if I'm missing something or if this was indeed fixed in serde

Reply:
> Btw many of the rpcs using `FixedU128` would have the same issue

That's not true, FixedU128 has a custom`Serialize` implementation that serialized as string (similarly to our balance wrapper)

Reply:
But if you tested then I guess they must have fixed it (about time, too)